### PR TITLE
feature/HEL-1396 create software artifact with unique name

### PR DESCRIPTION
**Pull request by daniellesarmiento**
Added a fallback logic for when: 
- there is a custom Software Artifact (SA) name 
- the name of SA is already taken

### CODE REVIEW

File: integration-api-default-config.json [hunk 1]
```diff
@@ -727,8 +727,14 @@
...
 						},
 						"values": [
+							{
+								"expr": "${lx.factsheet.name}"
+							},
 							{
 								"expr": "${integration.valueOfForEach}"
+							},
+							{
+								"expr": "${integration.valueOfForEach}(Copy ${integration.now.getYear()}-${integration.now.getMonthValue()<10?'0':''}${integration.now.getMonthValue()}-${integration.now.getDayOfMonth()<10?'0':''}${integration.now.getDayOfMonth()} ${integration.now.getHour()<10?'0':''}${integration.now.getHour()}:${integration.now.getMinute()<10?'0':''}${integration.now.getMinute()})"
```
Comment (SudharmaPuranikLeanIX) [inline integration-api-default-config.json:737]: just for my understanding.! Where can we find this expression template api? 
For Eg: `integration.now.getMinute()`

File: integration-api-default-config.json [hunk 2]
```diff
@@ -727,8 +727,14 @@
...
 							}
```
Comment (SudharmaPuranikLeanIX) [inline integration-api-default-config.json:738]: just for my understanding.! Where can we find this expression template api?. Is there documentation for this?
For Eg: integration.now.getMinute()

Comment (daniellesarmiento) [reply integration-api-default-config.json:738]: @SudharmaPuranikLeanIX https://docs.leanix.net/docs/advanced#using-integration-object

Comment (SudharmaPuranikLeanIX) [reply integration-api-default-config.json:738]: Thank you :) @daniellesarmiento

**Review by rajeshmohanty-leanix — APPROVED** [review]

**Pull request merged**